### DISCUSSION TITLE
Coordinator Failure

### DISCUSSION THREAD

**Discussion by jmachta** [Koenkk/zigbee2mqtt#23671] [Devices]
Hello All, 

I have a question regarding the situation, when for any reason coordinator fails - this is I guess standard issue of all Zigbee networks, as the coordinator is really a single point of failure. 

Somewhere, I read in a discussion, that Zigbee devices are able to talk to each other without coordinator (of course, after the network is established). So, I was wondering: when I have Z2M, it is using coordinator to gather data from Zigbee network and send it to control app & control, who can join network and who not. If the coordinator fails, it means, that control application (e.g. OpenHab) has no way how to talk to Zigbee network, however the network is still running, correct? 

Provided that devices are still able to talk to each other thru zigbee routers, would that be possible, that Z2M is connected to Zigbee network thru some other type of device? E.g. a router? Or is there any other way how to inject commands to the network (sort of alternative channel between OpenHab and devices? 

Sorry, I am not any expert in Zigbee and this is just a high level idea that arises in my mind. 

Another approach how to solve this issue is a clone of coordinator (and its address) so that it is possible to start another system with this coordinator connected => network is complete again... 

Thank you in advance...

**Comment by lux73** [comment]
when your Coordinator fails - your whole Zigbee Network will unreachable

compare it with Hue Bridge - if the Bridge is failing then the Zigbee Network via Bridge is also offline ^^

cloning a spare Coordinator should work BUT some Data is stored inside the Coordinator and if the System is not clean shutdowned than it could lead to some Problems because the everytime with clean shutdown backuped **coordinator_backup.json** is not up-to-date

**Reply by jmachta** [reply]
Thank you for the reply. Actually, I was not aware that there is a backup file. So, it means then, that two scenarios are possible:
1. If my dongle fails but underlying Linux works well, I can duplicate the backup file to second system, where cloned dongle is and start the network backup, right?
2. If complete PC fails, backup system has no way to take over the data and needs to start with old version of the file. What should I expect then? Extended stabilization period of Zigbee or could this lead also to inability of some nodes to communicate? 

What is the right way of shutting down the network? Is only shutdown of the host system down sufficient or do I need to power the devices off first?
